src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private final Random random = new Random();
+
+    public Ball getRandomBall() {
+        return new Ball(new ColorSupplier().getRandomColor(), random.nextInt(101));
```

Comment:
No need to instantiate `ColorSupplier` each time after calling the method `getRandomBall()`.

You can instantiate it once as you did with `Random`.